build/crds/build-crds.sh

```diff
@@ -15,7 +15,6 @@
 # limitations under the License.
 
 set -o errexit
-set -o nounset
```

Comment:
Should this be a different commit? Doesn't seem obviously related...

Reply:
It is because of the `-z` check and we now use unbound variables.